doc/build-windows.md

```diff
@@ -96,6 +96,9 @@ results in `Autoconf` configure scripts being able to execute Windows Portable E
 unexpected behaviour during the build, such as Win32 error dialogs for missing libraries. The recommended approach
 is to temporarily disable WSL support for Win32 applications.
 
+Note that if you are are running into the following errors while executing `make HOST=x86_64-w64-mingw32`, ensure that your Line endings are set to LF (Linux) and not CRLF (Windows) on the `config.guess` and `config.sub` files found in the depends folder.
+> ./config.sub: 80: Syntax error: word unexpected (expecting "in")
```

Comment:
Instead of `> ` for formatting, maybe using backticks, to make it clear it's in-line code. See also https://meta.stackexchange.com/questions/189920/what-is-the-3-x-backquote-markdown-used-for

Reply:
Thanks for the input! I think I'm missing something here :) - Allow me to be dumb for a second hehe: this is currently how the edit looks:

![image](https://user-images.githubusercontent.com/4400555/119684931-d2453c00-be3c-11eb-992d-3a242bee980b.png)

Would you prefer it look like this?

![image](https://user-images.githubusercontent.com/4400555/119684811-b772c780-be3c-11eb-8a64-df6066bcca6c.png)

If so. Cool :)

Reply:
I was thinking (B)

Reply:
B it is. I've also squashed. Thanks!